modules/ROOT/pages/configure-monitoring-cloudhub.adoc

```diff
@@ -53,7 +53,22 @@ anypoint-cli --username="..." --password="..."
 If you are using Maven for deployment, you must modify the `<muleVersion>`
 tag in the `pom.xml` file. For example, you can change `<muleVersion>3.9.3</muleVersion>` to `<muleVersion>3.9.4</muleVersion>`).
 
-== Enable Monitoring for apps deployed to CloudHub
+== Enable Monitoring for Apps Deployed to CloudHub by Default
+
+You can configure Anypoint Monitoring to enable monitoring by default for any newly deployed apps to CloudHub. When you enable the *(checkbox name)*, any Mule app that you deploy to CloudHub has monitoring enabled by default. 
+If you have a Titanium subscription to Anypoint Platform, this feature is enabled by default.
+
+. Sign in to Anypoint Platform and navigate to *Anypoint Monitoring*.
+. In the navigation menu, click *Settings*.
++
+image::monitoring-settings-screen.png[]
++
+. Select *CloudHub*.
+. Click *(checkbox name)*.
+
+If you deployed Mule apps to CloudHub before you enabled this feature, you must enable monitoring for those apps manually. This feature does not retroactively enable monitoring for previously deployed apps.
+
+== Enable Monitoring for Apps Previously Deployed to CloudHub
 
 You can enable monitoring for Mule apps running on non-AM versions of Mule in the Anypoint Monitoring *Settings* page, the application's *Settings* page, or by using the Anypoint CLI for continuous integration and delivery (CI/CD).
```

Comment:
The "non-AM" versions are no longer applicable. Let's take it out.

Reply:
If anyone is still behind on the times and running -AM versions, we should leave this. Otherwise, I'm happy to remove it.

Reply:
-AM was a suffix added to certain Runtime versions. It was always abbreviated in this manner. But I will follow up regarding edge cases of users who may or may not use them. We no longer support -AM runtimes at all, to my knowledge.